exaqute/__init__.py

```diff
@@ -2,18 +2,21 @@
 
 from .common import *  # noqa
 
-exacute_backend = os.environ.get("EXAQUTE_BACKEND")
-print("EXAQUTE_BACKEND=", exacute_backend)
+exaqute_backend = os.environ.get("EXAQUTE_BACKEND")
 
-if exacute_backend:
-    exacute_backend = exacute_backend.lower()
+if exaqute_backend:
+    print("EXAQUTE_BACKEND={}".format(exaqute_backend))
+else:
+    exaqute_backend = "local"
+    print("Default ExaQUte backend: {}".format(exaqute_backend))
+
+exaqute_backend = exaqute_backend.lower()
 
-if not exacute_backend:
-    print("ExaQUte backend: Local")
+if exaqute_backend == "local":
     from .local import *  # noqa
-elif exacute_backend == "hyperloom":
-    from .hyperloom import *  # noqa
-elif exacute_backend == "pycompss":
-    from .pycompss import *  # noqa
```

Comment:
Why are you removing `hyperloom` and `pycompss` imports?

Reply:
these lines shoudn't be removed.

Reply:
Indeed, that's a stupid mistake. This removal was copied from the `exaqute` module in XMC, which has only the `local` backend. I had replaced these with an error message.  Thanks for catching this, Riccardo!

I have fixed it in commit `17c5ede332aecf3109a08ea8526003bb8f652fb0`.